lib/client/twofa/pushtoken/pushtoken.go

```diff
@@ -94,6 +108,7 @@ func doGenericPushCheck(client *http.Client,
 	baseURL string,
 	pushType string,
 	userAgentString string,
+	codeIsDone chan bool,
```

Comment:
Minor nit: to prevent this function from writing to the channel, have the type be:
`<-chan bool`.